.github/dependabot.yml

```
@@ -5,3 +5,10 @@ updates:
     schedule:
       interval: "monthly"
     open-pull-requests-limit: 10
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      # Check for updates to GitHub Actions every week
+      interval: "weekly"
+    reviewers:
+      - "az-digital/developers"
```

Something I wish I would have added to the Quickstart config (and think we should still add there, too):
```suggestion
      - "az-digital/developers"
    labels:
      - "ci"
```